README.md

```diff
@@ -30,6 +30,27 @@ You can install `napari-micromanager` via [pip]:
 
     pip install napari-micromanager
 
+### Getting micromanager adapters:
+
+The easiest way to get the micromanager adapters is to use:
+
+```
+python -m pymmcore_plus.install
+```
+
+this will install micromanager to the pymmcore_plus folder in your site-package; use this to see where:
+
+```
+python -c "from pymmcore_plus import _util; print(_util.find_micromanager())"
+```
+
+alternatively, you can direct pymmcore_plus to your own micromanager installation with the `MICROMANAGER_PATH`
+environment variable:
```

Comment:
I think that it will also pick up the default install location? So see main comment, I think there's some value in also advertising that it works with the current MM setup.